src/vehicle_gui.cpp

```diff
@@ -2658,8 +2657,7 @@ struct VehicleDetailsWindow : Window {
 					WID_VD_DETAILS_TRAIN_VEHICLES,
 					WID_VD_DETAILS_CAPACITY_OF_EACH,
 					WID_VD_DETAILS_TOTAL_CARGO,
-					widget,
-					WIDGET_LIST_END);
+					widget);
```

Comment:
`widget` is one of the 4 widgets, it's not really needed to raise it twice.

Reply:
Good point. I've removed it.